db/migrate/20201008112050_add_content_to_article.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentToArticle < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :content, :text
+  end
+end
```

Comment:
Since this is not merged code and you have this migration only to add the content you can simply just remove this migration ( and the one you added to remove the column )